metric-schema-java/src/integrationInput/java/com/palantir/test/ReservedConflictMetrics.java

```diff
@@ -120,7 +120,7 @@ public IntBuilder long_(String long_) {
 
         @Override
         public void build(Gauge<?> gauge) {
-            registry.gauge(
+            registry.registerWithReplacement(
```

Comment:
the robots have recursed into an infinite loop

Reply:
I think this is a legitimate bug, i didn't handle all the cases I should have for generating registerWithReplacement

Reply:
Fix: https://github.com/palantir/metric-schema/pull/54